7.11.1 introduces typescript compile error with aggregate-error

#183 introduced a typescript compile break when it added a use of `AggregateError<T>` while still referencing `aggregate-error ^3.0.1`. 3.0.1 does not define a generic `AggregateError<T>` type, which leads octokit 7.11.1 consumers to encounter this typescript compiler error:

```
$ tsc
node_modules/@octokit/webhooks/dist-types/types.d.ts:31:51 - error TS2315: Type 'AggregateError' is not generic.

31 export interface WebhookEventHandlerError extends AggregateError<WebhookError> {
                                                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Found 1 error.
```

Please update project.json to depend on `aggregate-error ^3.1.0` to fix the break and release a new npm package.

--- a/package.json
+++ b/package.json
@@ -11,7 +11,7 @@
   "dependencies": {
     "@octokit/request-error": "^2.0.2",
     "@pika/plugin-ts-standard-pkg": "^0.9.2",
-    "aggregate-error": "^3.0.1",
+    "aggregate-error": "^3.1.0",
     "debug": "^4.0.0"
   },
   "devDependencies": {

